stable/kube-lego/values.yaml

```diff
@@ -5,6 +5,7 @@ config:
   ## Email address to use for registration with Let's Encrypt
   ##
   # LEGO_EMAIL: my@email.tld
+  LEGO_EMAIL: foo@bar.com
```

Comment:
> foo@bar.com

Should use `example.com` instead of `bar.com`. example.com is spec'd for this while bar.com is a live domain.

Reply:
Well this is news to me! Also there is actually a `foo@bar.com` https://bar.com/ That does need to get changed.

Reply:
See [RFC 6761](https://tools.ietf.org/html/rfc6761) for details. `example` in certain permutations is a reserved domain name.

Reply:
I changed it to `foo@example.com` - thanks!